.github/workflows/all_checks_osmosis.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         uses: actions/checkout@v3
       - name: Install Rust
         uses: dtolnay/rust-toolchain@stable
+      - name: Cleanup runner
+        run: sudo rm -rf /usr/local/share/boost && sudo rm -rf /usr/share/dotnet && sudo rm -rf /usr/local/lib/android && sudo rm -rf /opt/ghc && sudo rm -rf /opt/hostedtoolcache/CodeQL && sudo docker image prune --all --force
```

Comment:
Also, would sudo `docker system prune` be better in this context? So we also remove possible unused networks, volumes, etc.

Reply:
tried that too, didn't make a difference